include/bluetooth/mesh/sensor_types.h

```diff
@@ -0,0 +1,1106 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_sensor_types Sensor types
+ * @{
+ * @brief All available sensor types from the Mesh Device Properties
+ *        Specification.
+ */
+
+#ifndef BT_MESH_SENSOR_TYPES_H__
+#define BT_MESH_SENSOR_TYPES_H__
+
+#include <bluetooth/mesh/sensor.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @def BT_MESH_SENSOR_TYPE_FORCE
+ *
+ *  @brief Force the given sensor type to be included in the build.
+ *
+ *  If @ref CONFIG_BT_MESH_SENSOR_FORCE_ALL is disabled, only referenced sensor
+ *  types will be included in the build, and the node will be unable to
+ *  interpret the rest. Use this macro inside a function to force the type to be
+ *  included in the build:
+ *
+ *  @code{.c}
+ *  void some_function(void)
+ *  {
+ *      BT_MESH_SENSOR_TYPE_FORCE(bt_mesh_sensor_time_since_motion_sensed);
+ *      BT_MESH_SENSOR_TYPE_FORCE(bt_mesh_sensor_motion_threshold);
+ *  }
+ *  @endcode
+ *
+ *  @param[in] _type Sensor type to force into the build.
+ */
+#define BT_MESH_SENSOR_TYPE_FORCE(_type)                                       \
+	{                                                                      \
+		const struct bt_mesh_sensor_type *volatile __unused val =      \
+			&_type;                                                \
+	}
+/**
+ * @defgroup bt_mesh_sensor_types_occupancy Occupancy sensor types
+ * @{
+ */
+
+/**
+ * Motion sensed
+ *
+ * Channels:
+ * - Motion sensed
+ *   - Unit: Percent
+ *   - Encoding: 8 bit unsigned scalar (Resolution: 0.5 %)
+ *   - Range: 0 to 100.0
+ */
+extern const struct bt_mesh_sensor_type bt_mesh_sensor_motion_sensed;
+
+/**
+ * Motion threshold
+ *
+ * This sensor type should be used as a sensor setting.
+ *
+ * Channels:
+ * - Motion threshold
+ *   - Unit: Percent
+ *   - Encoding: 8 bit unsigned scalar (Resolution: 0.5 %)
+ *   - Range: 0 to 100.0
+ */
+extern const struct bt_mesh_sensor_type bt_mesh_sensor_motion_threshold;
+
+/**
+ * People count
+ *
+ * Channels:
+ * - People count
+ *   - Unit: _unitless_
```

Comment:
Isn't the unit people? :P

Reply:
Not according to the spec :smile: 
I think they didn't want to add a People unit